src/components/header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {
+  AppBar, Box, Toolbar,
```

Comment:
import elements should be on separate lines

Reply:
+